docs/UserGuide.md

```diff
@@ -29,7 +29,7 @@ Want to contribute to development? Check out the [**Developer Guide**](Developer
 
 ---
 
-**Table of Contents**
+### Table of Contents
```

Comment:
```suggestion
## Table of Contents
```

I think it should be the same level as Quick Start?

Reply:
Ok thank you!